cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java

```diff
@@ -553,19 +553,19 @@ private void _convertToWarProject(File warsDir, File pluginDir) {
 				ivyFile.delete();
 			}
 
-			StringBuilder depsContent = new StringBuilder();
+			StringBuilder toolingContent = new StringBuilder();
 
-			depsContent.append("dependencies {\n");
+			toolingContent.append("dependencies {\n");
```

Comment:
Perhaps `toolingContent.append("dependencies {" + System.lineSeparator());` for better platform compatibility?